packages/evolution-generator/src/tests/test_labels_generator.py

```
@@ -814,4 +804,251 @@ def test_merged_section_translations_merges_nested_dicts(capsys):
+            language,
+            section,
+            labels_output_folder_path,
+            translations,
+        )
+
+        file_path = LabelsGenerator.get_labels_file_path(
+            labels_output_folder_path=labels_output_folder_path,
+            language=language,
+            section=section,
+        )
+        with open(file_path, "r", encoding="utf-8") as f:
```

Ruff (0.15.12)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)